src/_guides/language/coming-from/js-to-dart.md

```diff
@@ -2607,113 +2756,87 @@ class Animal {
 }
 ```
 
-In JavaScript, this is a convention,
-but in Dart this is a full language feature
-enforced by the compiler. 
+JavaScript uses the hash as a convention.
+Dart's compiler enforces use of the underscore for this feature.
 
-A difference that should be noted is that in Dart,
-private members are not private to the class,
-but are private to the library,
-meaning that private members can still be accessed
-from code that is considered part of the same library.
-By default, this is possible anywhere in the same file,
-so private class members can still be accessed from code
-in the same file. While you can expand this library scope
-beyond a single file using the `part` directive,
-it's generally advised to [avoid doing so][] and is usually
-reserved for code generators. 
+Dart makes private members private to the library, not the class.
+This means that you can access private members from code in the same library.
+By default, Dart limits access to private class members to code in the same file.
+To expand the scope of a library beyond one file, add the `part` directive.
+When possible, [avoid using `part`][]. Reserve using `part` for code generators.
 
-[avoid doing so]: /guides/libraries/create-library-packages#organizing-a-library-package
+[avoid using `part`]: /guides/libraries/create-library-packages#organizing-a-library-package
 
 ### Late variables
 
-Assign the `late` keyword to class fields to
-indicate they are initialized at a later point,
-while remaining non-nullable. This is useful
-for cases where a variable is never observed
-before being initialized, allowing it to be initialized later.
-This has several advantages over just labeling the field as nullable:
+To indicate that Dart initializes class fields at a later point,
+assign the `late` keyword to those class fields.
+Those class fields remain non-nullable.
+Do this when a variable cannot be observed before being initialized.
```

Comment:
This new sentence is confusing as every variable technically "cannot be observed before being initialized". I think the original text was trying to say something like "Do this when a variable does not need to be(or is not) observed/accessed right away and can be initialized later".

Reply:
Fair. Fixed.